pennylane_lightning/core/src/utils/TestKernels.hpp

```diff
@@ -22,9 +22,7 @@
 #include "TypeList.hpp"
 
 #include "cpu_kernels/GateImplementationsLM.hpp"
-#include "cpu_kernels/GateImplementationsPI.hpp"
 
 using TestKernels = Pennylane::Util::TypeList<
-    Pennylane::LightningQubit::Gates::GateImplementationsPI,
     Pennylane::LightningQubit::Gates::GateImplementationsLM, void>;
 #endif
```

Comment:
What about adding a blank line at the end of this file?

Reply:
Thanks!